libkbfs/journal_server.go

```diff
@@ -104,26 +110,26 @@ func makeJournalServer(
 	return &jServer
 }
 
-func (j *JournalServer) getDirLocked() string {
+func (j *JournalServer) rootPath() string {
+	return filepath.Join(j.dir, "v1")
+}
+
+func (j *JournalServer) tlfJournalPathLocked(tlfID TlfID) string {
 	if j.currentVerifyingKey == (VerifyingKey{}) {
 		panic("currentVerifyingKey is zero")
 	}
 
 	// Device IDs and verifying keys are globally unique, so no
```

Comment:
No, just verifying keys, which represent the device; this clause is justifying dropping the UID so no need to mention TLF IDs.